documentation/docs/migration-guide.md

```diff
@@ -9,6 +9,71 @@ import { DemoWrapper } from '@site/docs/components/DemoWrapper';
 
 This document is an overview of some important differences between Reveal 3.x and Reveal 4.x.
 
+## Entrypoint and import paths
+For Reveal 4 we changed into using a single entry point for everything: `@cognite/reveal`.
+So extensions and tools also should be imported from this entry point.
+We have however also added support for node's module `exports` such that if your tsconfig is setup with:
+```ts
+"moduleResolution": "Node16",
+```
+
+imports will still work the same way they did with Reveal 3.x. 
+Note that this may require a bump in typescript version.
+If this is not possible for you, then you need to change all imports to be `from @cognite/reveal` vs. f.ex. `from @cognite/reveal/tools`. 
```

Comment:
It would be good to show what old tools import looks like with a code snippet, and what new import looks like here and then explain that node16 can be used to use `@cognite/reveal/tools`